src/handler/http/models/mod.rs

```diff
@@ -16,4 +16,20 @@
 //! This module contains models that can be serialized and deserialized as JSON
 //! for HTTP responses and requests.
 
+pub mod alerts;
 pub mod folders;
+pub mod search;
+pub mod stream;
```

Comment:
No domain logic lives inside any of these modules. These modules are exclusively for defining the contract with the REST HTTP API and defining the JSON deserialization/serialization logic for that contract.